frontend/src/service/ApiService.js

```diff
@@ -0,0 +1,20 @@
+import axios from "axios";
+
+const productUrl = '/api/products'
+
+export const getProductsById = (id) =>
+    axios.get(`${productUrl}/${id}`).then(response => response.data)
+
+export const getUserProductsList = () =>
+    axios.get(`${productUrl}`).then(response => response.data)
```

Comment:
```suggestion
    axios.get(productUrl).then(response => response.data)
```

you can simply use your const here if you do not want to append anything